packages/bff/src/routers/agreementRouter.ts

```diff
@@ -132,7 +135,10 @@ const agreementRouter = (
           ctx
         );
 
-        return res.status(200).json(result).end();
+        return res
+          .status(200)
+          .json(bffApi.CatalogEServices.parse(result))
```

Comment:
is this `CompactEServicesLight`?

Reply:
You are right, unfortunately, since the models are compatible the compiler doesn't complain.